app/controllers/api/v2/qc_results_controller.rb

```diff
@@ -17,14 +17,22 @@ def create
         end
       end
 
-      def qc_results_params
+      def qc_results_params # rubocop:disable Metrics/MethodLength
```

Comment:
This is annoying when the tools fight each other isn't it?  Especially over a very restrictive rule.  I would argue that Rubocop is wrong here and the method is the same length as it always was.  You should be able to work around this limitation by declaring the permitted fields in a separate method:

```
def permitted_qc_results_params do
  :barcode, :uuid, :well_location, :key, :value, :units, :cv, :assay_type, :assay_version
end
```

Then using the method in the `permit` call:

```
ActionController::Parameters.new(p.to_unsafe_h).permit(permitted_qc_results_params)
```